assessments/guide/README.md

```diff
@@ -9,26 +11,109 @@ To provide the CNCF’s TOC with effective information about the security of dif
+     will recruit at least one additional reviewer, if one is not already
+assigned, and facilitate the process 
+### 3. Conflict of interest statement and review
+In order to remediate unfair advantage or ethical issues  all reviewers are
+required to provide a statement indicating all hard and soft conflicts they
+maintain prior starting the security assessment.
+   * **Lead security reviewer and additional security reviewers** comment any
+     conflict of interest in the project's assessment ticket using the below
+format:
+
+| Hard Conflicts | Y/N |
+| :------------- | :-: |
+| Reviewer is a currently a maintainer of the project |  |
+| Reviewer is direct report of/to a current maintainer of the project |  |
+| Reviewer is paid to work on the project |  |
+|Reviewer has significant financial interst directly ties to the success of the project |  |
+
+
+| Soft Conflicts | Y/N |
+|: ------------- | :-: |
+| Reviewer belongs to the same company/organization of the project, but does not work on the project |  |
+| Reviewer uses the project in his/her work |  |
+| Reviewer has contributed to the project |  |
+| Reviwer has a personal stake in the project (personal relationships, etc.) |  |
+   * The **lead security reviewer** will confirm all conflicts are specified and
+     escalate any conflict concerns, hard conflicts or multiple soft conflicts,
+to the **security assessment facilitator** for concurrence to proceed or if a
+reviewer is ineligible to participate.  
+### 4. Clarifying questions phase
+The clarifying questions phase is the responsibility of the **Lead security
+reviewer** to ensure it is complete.  They may delegate this task to another
+reviewer. This phase enables security reviewers to focus on the security and
+technical details of the project.  The clarifying question phase is conducted
+prior to the *3 week* timeframe for assessments.
+   * **Lead security reviewer or their designee** will perform inital,
+     clarifying review to:
+      * Verify completeness
+      * Ask for clarifications
+      * Ensure terms are defined
+      * Ensure concepts introduced are explained with context
+      * Provide quick feedback 
+### 5. Security review
+The security review provides time for the security reviewers and the project to
+address security and technical details associated with the project.  Information
+created or received out of the review is leveraged in making the final
+assessment.
+   * **Project** posts their document to the project security assessment
+     channel, allowing at least one week for review prior to Q&A
+   * **Security reviewers** review the self-assessment document and provide
+     comments and questions
+      * It is highly recommended that security reviewers familiarize themselves
+        with the project's repo and docs if available
+   * **Security reviewers and project lead/pocs** ensure all reviewer
+     questions/comments are addressed and finalize the self-assessment
```

Comment:
We need something in here about the summary document that the assessment team writes.  Usually, we've been doing a draft before the presentation to have some things to discuss at that time.  It isn't finalized until after the presentation and incorporates feedback from it.